projects/plugins/jetpack/extensions/blocks/subscriptions/controls.js

```diff
@@ -231,7 +231,15 @@ export default function SubscriptionControls( {
 					label={ __( 'Show subscriber count', 'jetpack' ) }
 					checked={ showSubscribersTotal }
 					onChange={ () => {
-						setAttributes( { showSubscribersTotal: ! showSubscribersTotal } );
+						setAttributes( {
+							showSubscribersTotal: ! showSubscribersTotal,
+							// Don't do anything if set previously, but by default set to false. We want to disencourage including social count as it's misleading.
+							// We don't want to rely setting "default" in block.json to falsy, because the default value was previously "true".
+							// Hence users without this set will still get social counts included in the subscriber counter.
+							// Lowering the subscriber count on their behalf with code change would be controversial.
+							includeSocialFollowers:
+								typeof includeSocialFollowers === 'undefined' ? false : includeSocialFollowers,
```

Comment:
To address @pkuliga your feedback at https://github.com/Automattic/jetpack/pull/34617#pullrequestreview-1841614648, I need to change the last `includeSocialFollowers` to something, maybe `false`?  My head isn't fresh enough now to wrap around this, and see what exactly needs to be here. Might be we even can't do anything about it since we want to preserve backwards compatibility and default to `false`? Tomorrow's problem!

Reply:
Actually I'll just merge and we can address in a follow-up!

Reply:
Sure, let's do this!